### ISSUE TITLE
Non alpha macros not being hightlighted 

### ISSUE THREAD

**Issue by smabie** [JuliaEditorSupport/julia-emacs#139]
![screenshot](https://user-images.githubusercontent.com/1126907/89943612-7f233500-dbec-11ea-8840-008290ae31f2.png)

You can see that the `@.` is not being highlighted, unlike like `@df` and `@pipe`. Is this intended behavior?

**Comment by non-Jedi** [comment]
No. You can probably change [this regex](https://github.com/JuliaEditorSupport/julia-emacs/blob/b5f5983d2b232c8bba4c5eff75cccdb787c19d98/julia-mode.el#L241) to just match `@` followed by any number of non-whitespace to fix it.
